platform/ios/src/MGLMapView.mm

```diff
@@ -5657,7 +5657,7 @@ - (void)setUserTrackingMode:(MGLUserTrackingMode)mode animated:(BOOL)animated co
     CLLocation *location;
     if (_userTrackingMode != MGLUserTrackingModeNone && (location = self.userLocation.location) && self.userLocationAnnotationView)
     {
-        [self locationManager:self.locationManager didUpdateLocations:@[location] animated:animated completionHandler:completion];
+        [self locationManager:self.locationManager didUpdateLocations:@[location] heading:nil animated:animated completionHandler:completion];
```

Comment:
same here, let's keep the original method as convenience, which internally calls the method with newly added `heading:` parameter.